### PR TITLE
Updated Clean and Save functions

### PR DESCRIPTION
**Pull request by Pav-Van**

### CODE REVIEW

File: workspace/data/process_data.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import sys
 import pandas as pd
+from sqlalchemy import create_engine
```
Comment (Pav-Van) [inline workspace/data/process_data.py:3]: Good

File: workspace/data/process_data.py [hunk 2]
```diff
@@ -23,12 +24,65 @@ def load_data(messages_filepath, categories_filepath):
 
 
 def clean_data(df):
```
Comment (Pav-Van) [inline workspace/data/process_data.py:26]: Love the change

**Comment by Pav-Van** [comment]
Good job

**Review by Pav-Van — COMMENTED** [review]
Good

**Review by balluout21 — APPROVED** [review]

**Pull request merged**